### PR TITLE
Bug 1676498 - Dates appear in the future in perfherder

### PR DESCRIPTION
**Pull request by ionutgoldan**
**Bug link:** https://bugzilla.mozilla.org/show_bug.cgi?id=1676498

### CODE REVIEW

File: ui/perfherder/helpers.js [hunk 1]
```diff
@@ -746,7 +746,9 @@ export const createGraphData = (seriesData, alertSummaries, colors, symbols) =>
       projectId: series.repository_id,
       id: `${series.repository_name} ${series.name}`,
       data: series.data.map((dataPoint) => ({
-        x: new Date(dataPoint.push_timestamp),
+        // Backend implicitly provides all dates as UTC.
+        // Let's make this explicit, so frontend doesn't get confused.
+        x: new Date(`${dataPoint.push_timestamp}Z`),
```
Comment (esanuandra) [inline ui/perfherder/helpers.js:751]: What is the `Z` for?

Comment (beatrice-acasandrei) [reply ui/perfherder/helpers.js:751]: The literal "Z" is actually part of the ISO 8601 datetime standard for UTC times. When "Z" (Zulu) is tacked on the end of a time, it indicates that that time is UTC, so really the literal Z is part of the time.

Comment (esanuandra) [reply ui/perfherder/helpers.js:751]: thanks!

**Review by esanuandra — APPROVED** [review]
It looks good to me!

**Review by beatrice-acasandrei — APPROVED** [review]

**Review by sarah-clements — APPROVED** [review]

**Pull request merged**